.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Signy",
+            "type": "go",
+            "request": "launch",
+            "mode": "exec",
+            "program": "${workspaceFolder}/bin/signy-debug",
+            "args": [
+                "sign",
+                "/home/radu/projects/src/github.com/cnabio/signy/testdata/cnab/bundle.json",
```

Comment:
TODO
Replace this with `${workspaceFolder}`

Reply:
This can go into your local .gitignore, no?

Reply:
Sure - but the intention was to add a document describing how to debug Signy.